webpack.config.js

```diff
@@ -35,7 +35,8 @@ const env = {
   MAINNET_DAPP_BASEURL: null,
   RINKEBY_DAPP_BASEURL: null,
   PROVIDER_URL: '',
-  REDUX_LOGGER: false
+  REDUX_LOGGER: false,
+  GA_TRACKING_ID: '' // must also be hard-coded in dev.html - this is used in components/analytics.js
```

Comment:
FYI, here is an example of how to set those env variables to different values in staging vs prod (needs separate origin-box PR):
See for example https://github.com/OriginProtocol/origin-box/pull/102

Reply:
According to @joshfraser we're using the same ID across all properties - thanks for the link tho - might be useful in future iterations